tests/data_asset/test_expectation_decorators.py

```diff
@@ -518,7 +519,7 @@ def expect_column_pair_values_to_be_different(
             "missing_count": 2,
             "unexpected_count": 3,
             "missing_percent": 40.0,
-            "unexpected_percent": 60.0,
+            "unexpected_percent": 100.0,
             "unexpected_percent_nonmissing": 100.0,
```

Comment:
Thank you very much for this :) I'd like to know more about why some values have changed and others have not?

Reply:
@Shinnnyshinshin Yes.  The previous values fail tests once the comparison has been fixed.